Source/ui_qt/CMakeLists.txt

```diff
@@ -232,6 +240,21 @@ set(QT_COMPONENTS Widgets Core)
 if(TARGET_PLATFORM_UNIX)
 	list(APPEND QT_COMPONENTS X11Extras)
 endif()
+
+set(QT_MOC_EXECUTABLE /usr/bin/moc)
```

Comment:
Note: only needed during when cross compiling, to avoid using target binary
TODO: add new flag `if(IS_CROSS_COMPILING)`